mfr/extensions/zip/templates/viewer.mako

```diff
@@ -0,0 +1,18 @@
+<link href="https://fonts.googleapis.com/css?family=Open+Sans:400,600,300,700" rel="stylesheet" type="text/css">
+<link rel="stylesheet" href="static/css/default.css">
+<div style="word-wrap: break-word;" class="mfrViewer">
+    <code>
+        <h1>
+            Zip File:
+        </h1>
+        <p>
+            Download the .zip file to view the contents.
+        <p>
+        <p>
+            ${zipped_filenames}
```

Comment:
My thoughts continued on @felliott comment about `infolist()` (not in the scope of this ticket), we can improve the UI (a friendly list instead of a raw text output) and add some meta data about them.